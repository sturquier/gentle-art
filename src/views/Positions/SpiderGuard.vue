<template>
  <h1>Spider guard</h1>
</template>
