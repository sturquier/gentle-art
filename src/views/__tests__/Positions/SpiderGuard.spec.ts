import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import SpiderGuard from '../../Positions/SpiderGuard/SpiderGuard.vue'

describe('SpiderGuard', () => {
  it('renders properly', () => {
    const wrapper = mount(SpiderGuard)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
