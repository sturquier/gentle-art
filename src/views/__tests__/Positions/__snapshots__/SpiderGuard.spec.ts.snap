// Vitest Snapshot v1

exports[`SpiderGuard > renders properly 1`] = `"<h1>Spider guard</h1>"`;
